tests/integration/bigquery.spec.ts

```diff
@@ -59,35 +59,51 @@ describe("@dataform/integration/bigquery", () => {
     const actionMap = keyBy(executedGraph.actions, v => v.name);
 
     // Check the status of the two assertions.
-    expect(actionMap.example_assertion_fail.status).equals(dataform.ActionExecutionStatus.FAILED);
-    expect(actionMap.example_assertion_pass.status).equals(
+    expect(actionMap["df_integration_test_assertions.example_assertion_fail"].status).equals(
+      dataform.ActionExecutionStatus.FAILED
+    );
+    expect(actionMap["df_integration_test_assertions.example_assertion_pass"].status).equals(
       dataform.ActionExecutionStatus.SUCCESSFUL
     );
 
     // Check the status of the two uniqueness assertions.
-    expect(actionMap.example_assertion_uniqueness_fail.status).equals(dataform.ActionExecutionStatus.FAILED);
-    expect(actionMap.example_assertion_uniqueness_fail.tasks[1].error).to.eql("Assertion failed: query returned 1 row(s).");
-    expect(actionMap.example_assertion_uniqueness_pass.status).equals(
-      dataform.ActionExecutionStatus.SUCCESSFUL
-    );
+    expect(
+      actionMap["df_integration_test_assertions.example_assertion_uniqueness_fail"].status
+    ).equals(dataform.ActionExecutionStatus.FAILED);
+    expect(
+      actionMap["df_integration_test_assertions.example_assertion_uniqueness_fail"].tasks[1].error
+    ).to.eql("Assertion failed: query returned 1 row(s).");
+    expect(
+      actionMap["df_integration_test_assertions.example_assertion_uniqueness_pass"].status
+    ).equals(dataform.ActionExecutionStatus.SUCCESSFUL);
 
     // Check the data in the incremental table.
-    let incrementalTable = keyBy(compiledGraph.tables, t => t.name).example_incremental;
+    let incrementalTable = keyBy(compiledGraph.tables, t => t.name)[
+      "df_integration_test.example_incremental"
+    ];
     let incrementalRows = await getTableRows(incrementalTable.target, adapter, dbadapter);
     expect(incrementalRows.length).equals(1);
 
     // Re-run some of the actions.
     executionGraph = await dfapi.build(
       compiledGraph,
-      { actions: ["example_incremental", "example_table", "example_view"] },
+      {
+        actions: [
+          "df_integration_test.example_incremental",
```

Comment:
let's keep these unqualified, please - we need that to continue working. (of course if the name is ambiguous then we should throw an error somewhere). (same elsewhere)